frontend/src/lib/components/PropertyFilters/PropertyFilters.scss

```diff
@@ -0,0 +1,20 @@
+@import '~/vars';
+
+.property-filters {
```

Comment:
Time for a `naming things` chat. We've been using two styles in parallel in the codebase

1. Component `SomeName`, css style `.some-name`
2. Component `SomeName`, css style `.SomeName` 

I initially used style 1 out of habit, though @Twixes at some point switched to not doing the extra step of changing the capitalization... and after a spiritual trip to the desert, I started to wonder as well: why do we add those dashes here?

So I propose we try to use option nr 2 in all new code... and as much as possible in the code we refactor as well. Possibly worth the effort to rename here as well? 🤷 

(Whether then inside those classes we use BEM, BOOM, KAPLOW or RZESZOV doesn't really matter for me)

Reply:
Agreed. I actually was gonna raise the same point. Main benefit of Nr 2 for me (other than consistency) is also quick searchability for finding all related code easily. I'll at least update all the things I've touched to follow Nr2